vector/src/main/res/values/strings.xml

```diff
@@ -2137,7 +2142,6 @@
     <string name="direct_room_profile_encrypted_subtitle">Messages here are end-to-end encrypted.\n\nYour messages are secured with locks and only you and the recipient have the unique keys to unlock them.</string>
     <string name="room_profile_section_security">Security</string>
     <string name="room_profile_section_restore_security">Restore Encryption</string>
-    <string name="room_profile_section_security_learn_more">Learn more</string>
```

Comment:
This is not the proper way to remove string. Please refer to this: https://github.com/vector-im/element-android/blob/main/CONTRIBUTING.md#removing-existing-strings